addons/survey/static/src/js/tour_test_survey.js

```diff
@@ -0,0 +1,198 @@
+odoo.define('survey.tour_test_survey', function (require) {
+'use strict';
+var Core = require('web.core');
+var Tour = require('web.Tour');
+var _t = Core._t;
+
+
+Tour.register({
+    id:   'test_survey',
+    name: _t("try to create and fill survey"),
+    path: '/survey/start/user-feedback-form-1/phantom',
+    mode: 'test',
+    steps: [
+        // Page-1
+        {
+            title:     "Clicking on Start Survey",
+            element:   'a.btn.btn-primary.btn-lg:contains("Start Survey")',
+        },
+        // Question: Where do you develop your new features?
+        {
+            title:     "Selecting answer 'Yes, I use a version < 7.0'",
+            element:   'select[name="1_1_1"]',
+            sampleText:'1',
+        },
+        // Question: Which modules are you using/testing?
+        {
+            title:     "Ticking answer 'Sales Management'",
+            element:   'input[name="1_1_2_5"][value="5"]',
+            sampleText:'5',
+        },
+        {
+            title:     "Clicking on Next Page",
+            element:   'button[value="next"]',
+        },
+        // Page-2
+        // Question: What do you think about the documentation available on doc.openerp.com?
```

Comment:
The question is from demo data and it contains 'doc.openerp.com'. If I will change here it will violate the consistency.